### PR TITLE
RA-1198 - Added indicators to html components

### PR DESCRIPTION
**Pull request by tmarzeion**
These minor changes are made to make testing easier by giving classes and id's for some components.
This PR is dependency for https://github.com/openmrs/openmrs-distro-referenceapplication/pull/150

### CODE REVIEW

File: omod/src/main/webapp/pages/allergies.gsp [hunk 1]
```diff
@@ -95,12 +95,8 @@ ${ ui.includeFragment("uicommons", "infoAndErrorMessage")}
 <br/>
 
 <% if (hasModifyAllergiesPrivilege) { %>
-    <button class="cancel" onclick="location.href='${ ui.escapeJs(returnUrl) }'">
-        ${ ui.message("uicommons.return") }
-    </button>
-	<button class="confirm right" onclick="location.href='${ ui.pageLink("allergyui", "allergy", [patientId: patient.id, returnUrl: returnUrl]) }'">
-	    ${ ui.message("allergyui.addNewAllergy") }
-	</button>
+    <input class="cancel" type="button" value="${ ui.message("uicommons.return") }" onclick="location.href='${ ui.escapeJs(returnUrl) }'">
```
Comment (rkorytkowski) [inline omod/src/main/webapp/pages/allergies.gsp]: Why did you change button to input?

Comment (tmarzeion) [inline omod/src/main/webapp/pages/allergies.gsp]: @rkorytkowski I've missed this change while making PR, sorry for that :)
